scripts-dev/complement.sh

```diff
@@ -32,10 +32,19 @@ if [[ -z "$COMPLEMENT_DIR" ]]; then
   echo "Checkout available at 'complement-master'"
 fi
 
+# If we're using workers, modify the docker files slightly.
+if [[ -n "$WORKERS" ]]; then
+  DOCKERTAG_SUFFIX=-workers
+  DOCKERFILE_SUFFIX=Workers
```

Comment:
@anoadragon453 asked for these to just be separate variables instead of suffixes.